src/zcl/definition/cluster.ts

```diff
@@ -3685,6 +3685,15 @@ const Cluster: {
         commands: {},
         commandsResponse: {}
     },
+    manuSpecificCentraliteHumidity: {
+        ID: 0xFC45,
+        manufacturerCode: ManufacturerCode.Centralite,
+        attributes: {
+            measuredValue: { ID: 0, type: DataType.uint16 },
```

Comment:
```measuredValue: {ID: 0, type: DataType.uint16},```, else lint complains

Reply:
Thanks!  I couldn't figure out why the check was failing.